twilio/base/obsolete.py

```diff
@@ -21,3 +21,23 @@ def new_func(*args, **kwargs):
         )
 
     return new_func
+
+
+def deprecated_method(new_func=None):
+    """
+    This is a decorator which can be used to mark deprecated methods.
+    It will report in a DeprecationWarning being emitted to stderr when the deprecated method is used.
+    """
+
+    def deprecated_method_wrapper(func):
+
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            msg = 'Function method .{}() is being deprecated'.format(func.__name__)
```

Comment:
`s/ being//`?

Reply:
?

Reply:
Should take out the word `being`. It _is_ deprecated.

Reply:
Fixed